pyserini/demo/msmarco.py

```diff
@@ -16,13 +16,27 @@
 
 import cmd
 import json
+import os
+import random
 
 from pyserini.search import SimpleSearcher
 from pyserini.dsearch import SimpleDenseSearcher, TctColBertQueryEncoder, AnceQueryEncoder
 from pyserini.hsearch import HybridSearcher
 
 
 class MsMarcoDemo(cmd.Cmd):
+    # load questions from dev-subset
+    script_dir = os.path.dirname(__file__)
+    rel_path = "../../tools/topics-and-qrels/topics.msmarco-passage.dev-subset.txt"
```

Comment:
This won't work once we install as a PyPI package.

Load questions using internal APIs:
https://github.com/castorini/pyserini/blob/master/tests/test_load_topics_qrels.py#L305